src/index.html

```diff
@@ -4,9 +4,78 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="../styles.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="month">
+      <div class="month__day start-wed  ">
```

Comment:
for example, it's November now
```suggestion
      <div 
        class="
          month 
          month--start-day-tue
          month--month-length-30
        "
      >
```
notice – I've used modifiers here with `--`, modifiers themselves contain only `-`

Reply:
If I use nth-child of month--month-length-30 I can only it get to work if I put month--month-length-30 in all of the following div's.